general/culture.md

```diff
@@ -1,46 +1,118 @@
 # Culture
 
-At source{d} we don't believe that being transparent, diverse, honest, etc can be called company culture. For us that's just general consensus of what a good company is like.
-
-We believe that for a company culture to be effective and attract the right people it needs to be unique and it needs to create as many detractors as lovers. Those who dislike your company culture should join a different company, that’s perfectly fine. But those who love it will stay through the hardest of times and truly know that this company is their own and that it stands for the things they believe in.
-
-For this reason we decided to aim for a unique company culture that many people will dislike, but one that we deeply believe in.
+At source{d} we don't believe that being transparent, diverse, honest, etc can be called company culture.
+For us that's just general consensus of what a good company is like.
+
+We believe that for a company culture to be effective and attract the right people
+it needs to be unique
+and it needs to create as many detractors as lovers.
+Those who dislike your company culture should join a different company,
+that’s perfectly fine.
+But those who love it will stay through the hardest of times
+and truly know that this company is their own
+and that it stands for the things they believe in.
+
+For this reason we decided to aim for a unique company culture
+that many people will dislike,
```

Comment:
This is a complement to company culture, should it be a different line? I'm trying to understand it.